docs/experiments/taint-propagators.md

```diff
@@ -0,0 +1,53 @@
+---
+slug: taint-propagators
+append_help_link: true
+description: "Taint propagators allow you to customize how taint is propagated."
+---
+
+# Taint propagators
+
+Consider the following Python code:
```

Comment:
I don't like this header much because here the spec and the example are intermixed. If we had a more formal spec of what a propagator is, then followed by an example, then it would make more sense to me.